service/src/main/resources/db/migration/V438__assistance_need_decision_no_end_date.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE assistance_need_decision
+    ADD COLUMN end_date_not_known BOOLEAN DEFAULT FALSE NOT NULL,
```

Comment:
default should be dropped afterwards, we now prefer always explicitly setting values and avoiding defaults